clipboard/android/src/main/java/com/capacitorjs/plugins/clipboard/Clipboard.java

```diff
@@ -0,0 +1,75 @@
+package com.capacitorjs.plugins.clipboard;
+
+import android.content.ClipData;
+import android.content.ClipDescription;
+import android.content.ClipboardManager;
+import android.content.Context;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
+
+public class Clipboard {
+    private static final String TAG = "Clipboard";
+
+    public enum ContentType {
+        STRING,
+        IMAGE,
+        URL
+    }
+
+    private Context context;
+    private ClipboardManager clipboard;
+
+    public Clipboard(Context context) {
+        this.context = context;
+        this.clipboard = (ClipboardManager) context.getSystemService(Context.CLIPBOARD_SERVICE);
+    }
+
+    public void write(String label, ContentType contentType, String content) {
+        ClipData data = null;
+        switch (contentType) {
+            case STRING:
+            case IMAGE:
+            case URL:
+                data = ClipData.newPlainText(label, content);
+                break;
+        }
```

Comment:
```suggestion
    public void write(String label, String content) {
        ClipData data = ClipData.newPlainText(label, content);
```
We don't need the switch if all 3 do the same, and we don't need the contentType on the write function neither